clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/api/KubernetesClientApiConverter.groovy

```diff
@@ -264,7 +291,7 @@ class KubernetesClientApiConverter {
     def kubernetesHttpGetAction = new KubernetesHttpGetAction()
     kubernetesHttpGetAction.host = httpGet.host
     kubernetesHttpGetAction.path = httpGet.path
-    kubernetesHttpGetAction.port = httpGet.port?.intVal ?: 0
+    kubernetesHttpGetAction.port = httpGet.port?.toInteger() ?: 0
```

Comment:
If this fails it will crash that run of the caching agent (not all ports in k8s are provided as ints). This should be wrapped in a try/catch at the very least.